internal/logger/logger.go

```diff
@@ -184,8 +186,11 @@ func isTerminal(w fdWriter) bool {
 	return isatty.IsTerminal(w.Fd())
 }
 
-func isCILogs() bool {
-	return os.Getenv("CI") == "true"
+func (l *FunLogger) isCILogs() bool {
+	if os.Getenv("CI") == "true" {
```

Comment:
Question: Is `CI` a common envvar to use? Should we switch to something longer?

Reply:
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables